lecture-loops.html

```diff
@@ -117,10 +119,13 @@
     <p>
       <b>Challenge Question:</b> Now that we have all the pieces, write a program that iterates through an array from the back to the front and prints out only the values at the odd indices.
     </p>
+
+    <pre><code>var myArray = [1,1,2,3,5,8,13,21,34,55];
+// your program should output: 55,21,13,5,3,1</code></pre>
```

Comment:
This are "all the odd values", but the question is "the values at the odd indices"!

I think both questions are valid and interesting. Should we just have both?